<launch>

      <node pkg="RC_controller"
            type="sign_reader.py"
            name="sign_reader" 
            output="screen">
      </node>

   	<node pkg="RC_controller"
         type="clue_publisher.py"
         name="clue_publisher" 
         output="screen">
      </node>

	<node pkg="RC_controller"
         type="clue_verification.py"
         name="clue_verification" 
         output="screen">
      </node>

   	<node pkg="RC_controller"
         type="ocr.py"
         name="ocr" 
         output="screen">
      </node>

      <node pkg="RC_controller"
         type="road_driving.py"
         name="road_driving" 
         output="screen">
      </node>

      <!-- Load the camera controller parameters -->
      <rosparam command="load" file="$(find RC_controller)/controllers/camera_controller.yaml" />

      <!-- Spawn the camera position controller -->
      <node name="controller_spawner" pkg="controller_manager" type="spawner"
            args="front_cam_joint_position_controller" ns="/quad" />
   
</launch>
